docs/cli.rst

```diff
@@ -42,6 +42,41 @@ Stream data from files in the `Karabo bridge
 <https://in.xfel.eu/readthedocs/docs/data-analysis-user-documentation/en/latest/online.html#data-stream-to-user-tools>`_
 format. See :doc:`streaming` for more information.
 
+.. code-block:: shell
+
+   karabo-bridge-serve-files /gpfs/exfel/exp/XMPL/201750/p700000/proc/r0005 4321
+
+.. program:: karabo-bridge-serve-files
+
+.. option:: --source <source>
```

Comment:
I think we can also have `.. option::` directives for the positional arguments (run & port) if you want. Not a high priority, though.

http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-option